src/language-markdown/printer-markdown.js

```diff
@@ -373,7 +374,7 @@ function genericPrint(path, options, print) {
           : group(
               concat([
                 align(
-                  " ".repeat(options.tabWidth),
+                  " ".repeat(4),
```

Comment:
I can't find the spec about indention for `footnoteDefinition`, but remark can't parse correctly with more/less space.